### PR TITLE
Added perspective_r example

### PR DESCRIPTION
**Pull request by mtiberghien**
Just added perspective_r example to the group Interactions techniques

### CODE REVIEW

**Comment by domoritz** [comment]
@jheer I think this is another cool example to show off the flexibility of Vega.

**Comment by mtiberghien** [comment]
image
A screenshot of added example

**Comment by mtiberghien** [comment]
Dear @jheer , I'm 46, having 3 children and a wife which was born and raised in Paris: I'm really used to get a no for an answer and not take offense off it (well with few exceptions, :D). I noticed that you prefer an explicit silence to a no, and if I can understand the reasons, I find it is counter productive probably because of the guessing and waiting part. I'd be okay if you don't want of that 3D sample and I'll close the pull request without any bad feelings but I'd prefer if you'd say so in a few words, not having to decide by myself. I totally understand that one might be reluctant to show 3D representations when 2D representations often offer a better perspective but:

- Since I started studying (currently studying a Master Big Data at Paris8 university) machine learning algorithms, I see 3D representations all the time
- It's a great exercice to understand and explain how data update works in Vega
- Vega is a visualization grammar: in my opinion it's always good to demonstrate that you can use it with a large variety of expressions
- 3D is useless but nice to watch: it's beautiful how simple formulas can shape the virtual world. I came to computing (I'm a former circus acrobat) because I'd like to type on a keyboard (learning to type without watching the keyboard). I think that people might want to start to work with visualization because they enjoy to watch them

Have a nice day!

**Comment by hydrosquall** [comment]
It never crossed my mind to try visualizing 3d structures with Vega, amazed to see that it's possible and interaction-friendly! Thanks for sharing this example @mtiberghien !

File: docs/_data/examples.json [hunk 1]
```diff
@@ -102,7 +102,8 @@
         {"name": "interactive-legend"},
         {"name": "stock-index-chart"},
         {"name": "pi-monte-carlo"},
+        {"name": "perspective-r"},
         {"name": "pacman"},
-        {"name": "platformer"}
+        {"name": "platformer"}
```
Comment (domoritz) [inline docs/_data/examples.json:107]:
```suggestion
        {"name": "platformer"}
```

Comment (mtiberghien) [reply docs/_data/examples.json]: done

**Comment by mtiberghien** [comment]
Nice talk!

**Pull request closed without merging**